worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -117,7 +117,12 @@ private[celeborn] class Worker(
       Utils.fromCelebornConf(conf, TransportModuleConstants.PUSH_MODULE, numThreads)
     val pushServerLimiter = new ChannelsLimiter(TransportModuleConstants.PUSH_MODULE, conf)
     val transportContext: TransportContext =
-      new TransportContext(transportConf, pushDataHandler, closeIdleConnections, pushServerLimiter)
+      new TransportContext(
+        transportConf,
+        pushDataHandler,
+        closeIdleConnections,
+        pushServerLimiter,
+        conf.clientHeartbeatEnabled)
```

Comment:
it's also useful to help client reused by many tasks, because connection maybe idletimeout without heartbeat from worker acorssing different thread clients( client instance is process level).